website_product_availability/models/product.py

```diff
@@ -151,11 +151,12 @@ def _get_planned_dates(self):
     @api.multi
     def _product_availability_warehouse(self, warehouse_id):
         # search avalailability for stokable products
-        if self.type not in('product'):
+        if self.type not in 'product':
```

Comment:
el in es para comparar iterables, haber usado una tupla no está mal.... o usas == o usas in con un iterable. ambas opciones son válidas, tu cambio está incorrecto.